### DISCUSSION TITLE
Is this solution applied to Virtualbox Windows VM?

### DISCUSSION THREAD

**Discussion by Winson-Huang** [winapps-org/winapps#24] [Support] [answered]
Basic behind this solution is RemoteApp and RDP, these seems independent to KVM / QEMU, so is this solution applied to Virtualbox Windows VM too?

**Comment by oskardotglobal** [comment, marked as answer]
There's no reason it shouldn't work with the necessary changes, and one of the rewrite's features is to be able to just specify RDP connection details without having the VM created by the program. However, it is recommended to use qemu since hardware virtualization is so much faster than what Virtualbox offers.

For now, it is recommended to use [quickemu](https://github.com/quickemu-project/quickemu) or [Gnome Boxes](https://wiki.gnome.org/Apps/Boxes) to create and manage your VM.

**Comment by LDprg** [comment]
Technically the vm could be on another machine too (if you dont care about latency)
